### ISSUE TITLE
Decide what to do with "privacy" and "terms" links.

### ISSUE THREAD

**Issue by jdhtwo** [mattcroce/HSver1#32]
Bootstrap came with these in the footer. They look official, but since we don't have anything to offer in these areas, I commented them out for the moment.

We could simply make a privacy + terms page and put some filler text in there. Dunno.

**Comment by mattcroce** [comment]
Yeah. I think we should basically steal a boilerplate privacy statement from anywhere online. Something that just says, "when you come to this site you agree to your data being captured, tracked, etc." something like that. I would think we could do something similar with terms. "this site is intended for educational and entertainment purposes and in no way should it be construed as legal or financial advice, blah blah blah". kind of a CYA thing, but necessary i think.

**Comment by jdhtwo** [comment]
Weird. In my efforts to externalized my brain to Github, I already created this issue in #28! Closing #28 and keeping our comments here.
